docs/tutorials/tutorial-jupyter-docker.md

```diff
@@ -193,6 +207,13 @@ as well as the [Python client for Druid](tutorial-jupyter-index.md#python-api-fo
 
 You should now be able to access and complete the tutorials.
 
+## Tutorial setup for ARM-based devices
+1. Start Druid. You can use the [Quickstart (local)](./index.md) instance. The tutorials
+   assume that you are using the quickstart, so no authentication or authorization
+   is expected unless explicitly mentioned.
+2. Start Kafka and Jupyter using the command from [Start Kafka and Jupyter](#start-kafka-and-jupyter).
+3. Before starting a tutorial, open the notebook and edit the `host` variable to `"host.docker.internal"`.
```

Comment:
We'll need to update the notebooks to use the host variable. Not necessary in this PR. However, this instruction applies to the ###Start only the Jupyter container section.  Can you add it there, too?